supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -437,6 +437,22 @@ _Build an Operator using the Operator SDK._
 
 *See also*: xref:action[action]
 
+[discrete]
+[[quick-start]]
+==== quick start (noun)
+*Description*: There are two types of quick starts in OpenShift:
+
+* Quick starts that are guided tutorials in the web console
+* Quick start templates that allow users to quickly get started creating a new application
```

Comment:
s/allow/enable

Reply:
enable or permit, instead of allow.
to quickly get started... hmm.  ... to start creating new applications quickly?

Reply:
Thanks, updated to enable and reworded some!